Codykilpatrick/Celestia
# feat(database): create migrations for tables

These migrations create the following tables:
-market_history_pull
-market_order_pull
-model_predict_average_increase


**Test plan (required)**
1. Have Postgres installed and running in the background:
2. open a terminal and run the command `createdb celestia`
3. `cd packages/celestia-database`
4. run `npx db-migrate up` to get the migrations in.
5. `yarn db:connect` to launch the graphql interface
6. navigate to: `http://[::1]:5678/graphql` To view the graphQL gui.

If you open the explorer tab you should see all the new queries and migrations on the new tables

The next step is creating scripts that can seed the tables. I think that the static info like types and ids can be CSV files but also need to figure out migrations or scripts im not sure the best approach to add the market pull data to the db.

#22 needs to be merged before this one can be.
